tests/src/Integration/Event/UserLoginTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Event\UserLoginTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Event;
+
+/**
+ * Checks that the event "rules_user_login" is correctly defined.
+ */
+class UserLoginTest extends EventTestBase {
+
+  /**
+   * Tests the event metadata.
+   */
+  public function testUserLoginEvent() {
+    $event = $this->eventManager->createInstance('rules_user_login');
```

Comment:
Not sure there is much value in that. Rather than, maybe it makes sense to trigger the event with a real case and assert the event & defined contexts are there and accessible?